README.md

```diff
@@ -0,0 +1,51 @@
+# dgraph-schema
+dgraph-schema is node a CLI allowing you to manage your Dgraph schemas with ease.
+You can:
+- Print your current Dgraph schema
+- Print the output between your new schema & the current one
+- Alter your Dgraph schema if there are no conflicts
+
+## Requirement
+Node v13.12.0 or later
+New schema in file named `schema.js` in the same folder as `cli.js`
```

Comment:
A globally installed CLI will not be in the same folder, schema file have to be passed as an optional argument, or fallback by default to the command execution path